wso2is/hieradata/dev/wso2/wso2is/pattern-1/README.md

```diff
@@ -1,6 +1,7 @@
 # WSO2 Identity Server Pattern-1
 
-WSO2 Identity Server Deployment Pattern 1 runs two Identity Server instances fronted with a load balancer. Identity 
+[WSO2 Identity Server Deployment Pattern 1](https://docs.wso2.com/display/IS540/Deployment+Patterns#DeploymentPatterns-Pattern1-HAclustereddeploymentofWSO2IdentityServer) 
+runs two Identity Server instances fronted with a load balancer. Identity 
```

Comment:
I do not think that we deal with LBs at this level. So, do we need to mention that here ?

Reply:
Well, this is added to give a brief explanation on the deployment pattern that the respective hiera of the module is referring. Yes we don't need to talk about load balancers. But, a person should know which component in which deployment pattern can be configured over this hiera file.
So if you just go below on the same readme you can see, that after explaining the deployment pattern we have explained which component of this deployment pattern can be configured with the hiera yaml file.
Quoting:
"The hiera YAML in this directory can be used to setup each WSO2 Identity Server node participating in the cluster."

Reply:
When you say "Deployment Pattern 1 runs two Identity Server instances fronted with a load balancer", that might imply that we configure a LB as well here which is not the case, so let's just change the wording here.

Reply:
I don't agree. It clearly says the use of hiera yaml file below.
Also, if you go to the root readme there is a clear indication of what can be acheived over this repository artifacts.
Quoting:
"Note that this guide, explains only about setting up wso2is or wso2is_analytics puppet modules with respect to the deployment pattern selected. Setting up the infrastructure, load balancers and sharing runtime deployment artifacts among the nodes participating in a deployment is not in the scope of this guideline. Refer WSO2 Identity Server Deployment Guidelines to understand how to do a complete deployment of WSO2 Identity Server based on a standard deployment pattern."

Reply:
Make sense. +1